pages/1.11/release-notes/1.11.8/index.md

```diff
@@ -0,0 +1,97 @@
+
+# Issues fixed in DC/OS 1.11.8
+
+## Docker containers
+- COPS-4087	- For applications that use Docker containers with a Virtual IP address, backend port mapping resolves access to the application by using the `host_IP:port_number` instead of the `container_ip:port_number`.
+
+## GUI
+- COPS-3369 - Updates to the DC/OS UI provide better rendering for elements such as environment variables, secrets, labels, and version information.   
+- DCOS_OSS-1961	- Pod lifecycle stages are supported in the DC/OS UI.
+
+## Marathon
+- COPS-3764	- The upgrade to Marathon 1.6.x enables successful secret validation for secrets included in a Marathon JSON app definition file.
+- MARATHON-8317	- This release adds support for enhancements and issues fixed in Marathon 1.6.x. 
+
+## Mesos
+- COPS-3953	- The Mesos fetcher process automatically retries downloading files using their associated URI if the previously-downloaded and cached versions of the files are not found.
+- DCOS-43544 - Logic changes enable nested containers to run under the same user account as the user associated with their parent container by default. For nested containers in a pod, the default executor’s user--that is, the user running the top-level container--has been the framework user. In a scenario where the framework user is a normal user but the nested container user is `root`, the change in this release enables the second-level nested containers to run as the same user--for example, the `root` user--as the parent top-level container instead of as the framework user by default.
+- DCOS-43593 - This release fixes an issue that could cause Mesos master endpoints—-such as `reserveResources` or `createVolume`-—to fail during authorization. For example, before implementing this fix, the authorization requests for an endpoint might fail or be incomplete if there’s extreme load on the IAM service. The change in this release ensures that authorization requests for an endpint are complete before continuing.
+- DCOS-43670, DCOS-44827 - Updates to the `cgroups` event listener code that is used to poll events for a container ensure that the listener closes the file descriptor after the read operation is complete. This change prevents a race condition that can leave the container in an ISOLATING or PROVISIONING state.
+
+## Metronome
+- COPS-3573, DCOS-45564, DCOS_OSS-2535 - This release adds support for enhancements and issues fixed in Metronome 0.4.5.
+- DCOS_OSS-3616	- Metronome initialization improvements prevent Metronome from being in an incomplete state that could cause Mesos offers to be held in reserve.
+
+## Network
+- COPS-3924, DCOS_OSS-1954	- The distributed layer-4 load-balancer (`dcos-l4lb`) network component waits to route traffic until application scaling or health check is complete. 
+- COPS-4034, DCOS_OSS-4398	- Changes in this release prevent `dcos-net` from continously restarting `systemd-networkd` on a bare-metal server with bond interfaces.
```

Comment:
I would just say "Prevent..." instead of "Changes in this release"

Reply:
Can we keep it as
This release ...